source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -466,7 +469,8 @@ Pass::Status AggressiveDCEPass::ProcessImpl() {
   // TODO(greg-lunarg): Handle additional capabilities
   if (!get_module()->HasCapability(SpvCapabilityShader))
     return Status::SuccessWithoutChange;
-  // Current functionality assumes logical addressing only
```

Comment:
So, there's no actual way of testing for relaxed logical addressing?

Reply:
There is not quick check that can be done at the start of the pass.  However, it is check and enforced through the assert that are done in the `GetPtr` calls.